### PR TITLE
Contributor: Add missing interface definition

### PR DESCRIPTION
**Pull request by VadymSachenko**
## PR Description
Original contributor PR: https://github.com/spryker/spryker-docs/pull/1266
TBD

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/scos/dev/back-end-development/extending-spryker/development-strategies/project-modules/add-a-new-module.md [hunk 1]
```diff
@@ -170,7 +170,21 @@ class HelloWorldFacade extends AbstractFacade implements HelloWorldFacadeInterfa
...
-4. Modify the controller so that it calls the method added to your facade:
+```php
+<?php
+
+namespace Pyz\Zed\HelloWorld\Business;
+
+interface HelloWorldFacadeInterface
+{
+    /**
+     * @return string
```
Comment (fscholzspryker) [inline docs/scos/dev/back-end-development/extending-spryker/development-strategies/project-modules/add-a-new-module.md:181]: Do be very precise (at least in the documentation we should be) we should add the @api tag and the Specification as well

**Review by fscholzspryker — CHANGES_REQUESTED** [review]

File: docs/scos/dev/back-end-development/extending-spryker/development-strategies/project-modules/add-a-new-module.md (hunk 1 above)
Comment (VadymSachenko) [reply docs/scos/dev/back-end-development/extending-spryker/development-strategies/project-modules/add-a-new-module.md:181]: @fscholzspryker do we someone's assistance here to fix this? I'm no sure what to add and where.

Comment (fscholzspryker) [inline docs/scos/dev/back-end-development/extending-spryker/development-strategies/project-modules/add-a-new-module.md:181]:
```suggestion
     * Specification:
     * - Returns a salutation text
     *
     * @api
     *
     * @return string
```

Comment (fscholzspryker) [reply docs/scos/dev/back-end-development/extending-spryker/development-strategies/project-modules/add-a-new-module.md:181]: I added a suggestion - here you find also another example from core: https://github.com/spryker/event/blob/master/src/Spryker/Zed/Event/Business/EventFacadeInterface.php#L18

**Comment by VadymSachenko** [comment]
@jkreller thank you for you contribution into our project. Your changes have been approved and merged successfully within this PR.

**Review by lenadoc — APPROVED** [review]

**Review by fscholzspryker — APPROVED** [review]

**Comment by VadymSachenko** [comment]
@jkreller thank you for you contribution. Your changes have been approved and merged successfully.

**Pull request merged**